SPECS/xapi-test-utils.spec

```diff
@@ -0,0 +1,49 @@
+Name:           xapi-test-utils
+Version:        0.9.0
```

Comment:
As discussed "offline", I'd like to declare it 1.0.0 once we have confirmed that all stuff still works after this refactoring exercise, but it is hard to do that without having an RPM in the first place (chickens, eggs, etc). This will hopefully be very very soon.